docs/user/spark/architecture.rst

```diff
@@ -0,0 +1,36 @@
+Architecture
+------------
+
+GeoMesa Spark provides capabilities to run geospatial analysis jobs on
+the distributed, large-scale data processing engine `Apache Spark`_. It provides
```

Comment:
Yeah, I meant GeoMesa (or the GeoMesa Spark libraries, specifically), and I didn't want to start three sentences in a row with "GeoMesa Spark is ...". I've changed it to something less ambiguously worded.